### PR TITLE
Add tutorial on exporting for inference

### PR DESCRIPTION
**Pull request by hanlint**
This PR adds a notebook tutorial on exporting models for inference across three formats (ONNX, torchscript, and torch.fx)

Coming along for the ride: fix metadata on getting started so that syntax coloring on our docs site works properly.

### CODE REVIEW

File: examples/exporting_for_inference.ipynb [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+   "cell_type": "markdown",
+   "id": "7b755249",
+   "metadata": {},
+   "source": [
+    "# 🥡 Exporting for Inference\n",
+    "\n",
+    "Composer models are also `torch.nn.Module`, and thus can be exported like any other PyTorch module. In this tutorial, we walk through how to export your models into various common formats: [ONNX](https://onnx.ai/), [TorchScript](https://pytorch.org/docs/stable/jit.html), and [torch.fx](https://pytorch.org/docs/stable/fx.html). For more detailed options and configuration settings, please consult the linked documentation.\n",
+    "\n",
+    "## Algorithm compatibility\n",
+    "Some of our algorithms alter the model architecture in ways that may render them incompatible with some of the export procedures above. For example, BlurPool replaces some instances of `Conv2d` with `BlurConv2d` layers which are not compatible with `torch.fx` as they have data-dependant codepath. \n",
```
Comment (dskhudia) [inline examples/exporting_for_inference.ipynb:13]: nit: data-dependent codepath => data-dependent control-flow.

File: examples/exporting_for_inference.ipynb [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+    "\n",
+    "The following table shows which algorithms are compatible with which export formats for inference.\n",
+    "\n",
+    "|                        | torchscript | torch.fx | ONNX |\n",
+    "|------------------------|-------------|----------|------|\n",
+    "| apply_blurpool         | X           |          | X    |\n",
```
Comment (dskhudia) [inline examples/exporting_for_inference.ipynb:19]: Can we do check marks and crosses in this table? Currently crosses come across as algorithms not working with the corresponding export format.

Comment (hanlint) [reply examples/exporting_for_inference.ipynb]: Done!

File: examples/exporting_for_inference.ipynb (hunk 1 above)
Comment (hanlint) [reply examples/exporting_for_inference.ipynb]: done!

**Review by dskhudia — APPROVED** [review]
Thanks for adding this!

**Pull request merged**